git/git.go

```diff
@@ -190,12 +190,16 @@ func (provider *repositoryProvider) dumpFile(file *object.File, outputPath strin
 		filePathToCheck = strings.ToLower(filePathToCheck)
 	}
 
-	if len(provider.includePatterns) > 0 && !matches(filePathToCheck, provider.includePatterns) {
+	doNotSkip := false
```

Comment:
`doNotSkip` is confusing, can you reverse it to `skip`?

Reply:
sure